### ISSUE TITLE
Move JPD to it's own variable

### ISSUE THREAD

**Issue by Randyfreak** [haoyangnz/msh-kiwiplan#7]
The JPD variable comes from JPJ, it should be a stand alone variable. (Used when doing jid)

**Comment by haoyangnz** [comment]
Good idea. Let's do this after msh-core supports auto mshrc template update.
